docs/src/manual.md

```diff
@@ -0,0 +1,19 @@
+# Methods
+
+```@docs
+LinearInterpolation
+QuadraticInterpolation
+LagrangeInterpolation
+AkimaInterpolation
+ConstantInterpolation
+QuadraticSpline
+CubicSpline
+BSplineInterpolation
+BSplineApprox
+```
+
+# Utility Functions
+
+```@docs
+DataInterpolations.bracketstrictlymontonic
```

Comment:
We should update to make this use https://github.com/SciML/FindFirstFunctions.jl